dist/log.py

```diff
@@ -86,6 +86,19 @@ def printf_setup(f, i, nl_indent):
 def n_setup(f):
     return len(field_types[f[0]][4])
 
+# Check for an operation that has a file id type. Redact any user data
+# if the redact flag is set, but print operations for file id 0, known
+# to be the metadta.
```

Comment:
Spelling: `metadta`

Reply:
Fixed. And I updated the `fileid == 0` to `fileid == WT_METAFILE_ID` that I saw with fresh eyes while in there.